cluster/images/hyperkube/Makefile

```diff
@@ -21,7 +21,7 @@ REGISTRY?=gcr.io/google-containers
 ARCH?=amd64
 HYPERKUBE_BIN?=_output/dockerized/bin/linux/$(ARCH)/hyperkube
 
-BASEIMAGE=gcr.io/google-containers/debian-hyperkube-base-$(ARCH):0.4
+BASEIMAGE=gcr.io/google-containers/debian-hyperkube-base-$(ARCH):0.5
```

Comment:
FYI - will conflict with https://github.com/kubernetes/kubernetes/pull/52744

Reply:
yeah, I already reached out to @rphillips to sequence